lib/qless/worker/base.rb

```diff
@@ -166,9 +170,9 @@ def try_complete(job)
         #   - The job is being worked on by another worker
         #   - The job has been cancelled
         #
-        # We don't want to (or are able to) fail the job with this error in
+        # We don't want to (or are able to) fail the job with this warning in
```

Comment:
The "error" in this sentence was a reference to the Exception being rescued above, not to the log level used below.  As such, I think it's less confusing to revert the change in this line.